packages/button/src/vaadin-button-mixin.js

```diff
@@ -74,6 +74,11 @@ export const ButtonMixin = (superClass) =>
     _onKeyDown(event) {
       super._onKeyDown(event);
 
+      // Analogous to native button, we should trigger click event when modifier key pressed.
```

Comment:
Should this comment say `we should _not_ trigger click event when modifier key pressed`

Reply:
Ah, nevermind.